components/LiveFeed/Contribution/ContributionHeader.tsx

```diff
@@ -199,31 +205,137 @@ const ContributionHeader = ({ entry }: Args) => {
             {createdBy.authorProfile.isVerified && (
               <VerifiedBadge height={18} width={18} />
             )}
-
             {actionLabel}
+            <span className={css(styles.dot)}>•</span>
+            {
+              <div className={css(styles.secondaryText, styles.date)}>
+                {timeSince(item.createdDate)}
+              </div>
+            }
             {/* @ts-ignore */}
             {unifiedDocument && (
-              <span className={css(styles.unifiedDocument)}>
-                <ALink
-                  overrideStyle={styles.link}
-                  href={getUrlToUniDoc(unifiedDocument)}
-                >
-                  {truncateText(unifiedDocument?.document?.title, 100)}
-                </ALink>
-              </span>
+              <>
+                <span className={css(styles.dot)}>•</span>
+                <span className={css(styles.unifiedDocument)}>
+                  <ALink
+                    overrideStyle={styles.link}
+                    href={getUrlToUniDoc(unifiedDocument)}
+                  >
+                    {truncateText(unifiedDocument?.document?.title, 100)}
```

Comment:
not super opinionated here, but 100 characters seems like a lot, and causes the title to wrap on my 13" MacBook:

<img width="827" alt="Screenshot 2023-11-29 at 2 29 34 PM" src="https://github.com/ResearchHub/researchhub-web/assets/16143968/d8939259-ed0c-4f46-88cc-fa4ad1df5c1c">

Reply:
Good point, thanks!